modules/core/src/lib/layer.js

```diff
@@ -476,7 +476,6 @@ export default class Layer extends Component {
       model.id = this.props.id;
       model.program.id = `${this.props.id}-program`;
       model.geometry.id = `${this.props.id}-geometry`;
-      model.setAttributes(this.getAttributeManager().getAttributes());
```

Comment:
`_updateState` already set attributes from `attributeManager.getChangedAttributes()` which contains attributes created by the transition manager.

This line was overwriting the transition attributes with the original ones.

Reply:
OK, but I assume the removed line of code is not new in v6. Something else has changed? Maybe not worth digging in...

Reply:
I think this has always been excessive.